graders/python/requirements.txt

```diff
@@ -1,26 +1,26 @@
 beautifulsoup4==4.14.2
-bokeh==3.7.3
+bokeh==3.8.0
 colormath==3.0.0
 defusedxml==0.7.1
-Faker==37.8.0
+Faker==37.12.0
 folium==0.20.0
 ipython==8.37.0
-matplotlib==3.10.3
+matplotlib==3.10.7
 nbconvert==7.16.6
 nbformat==5.10.4
 networkx==3.4.2
-nltk==3.9.1
+nltk==3.9.2
 numpy==1.26.4
 openpyxl==3.1.5
 pandas==2.3.3
-Pillow==11.3.0
+Pillow==12.0.0
```

Comment:
https://github.com/python-pillow/Pillow/releases/tag/12.0.0: this has a few removals and deprecations, but they seem unlikely to impact questions.